lit_gpt/args.py

```diff
@@ -25,6 +25,8 @@ class TrainArgs:
     """Limits the number of optimizer steps to run."""
     max_seq_length: Optional[int] = None
     """Limits the length of samples. Off by default"""
+    tie_embeddings: Optional[bool] = None
```

Comment:
This arg needs to be inserted into all the existing configs, right?

Reply:
Why if the default is disabled? not-set aka None

Reply:
Shouldn't the configs reflect all args? In general we don't know whether defaults have or will change, and configs would be better if they were reproducible? So I would make it consistent even if the probability of it changing is low.

Reply:
I wanted to regenerate all configs with comments included when we stop moving things around and we have docstring for most of them. I can add this one now anyways